app/Imports/Srd/SrdRoutesImport.php

```diff
@@ -65,7 +66,9 @@ private function levelValid(?string $level): bool
 
     private function getNoteIds(string $notes): array
     {
-        return explode(self::NOTES_DELIMETER, substr($notes, 6));
+        return SrdNote::whereIn('id', explode(self::NOTES_DELIMETER, substr($notes, 6)))
```

Comment:
Forgive me - what's the significance of the 6? Comment might be useful :D

Reply:
The notes are in the format `Notes: X - Y`, so 6 starts the `substr` at the end of `Notes:` :D

Comment added!